receiver/receiver.go

```diff
@@ -47,6 +47,8 @@ func ShouldNotify(oldStatus int, newStatus int) bool {
 	switch newStatus {
 	case service.ALIVE:
 		return true
+	case service.DRAINING:
```

Comment:
Pretty sure I tried to keep these in the order of the `iota` in the `const`. Draining should go last IMO